src/components/ChatLog.jsx

```diff
@@ -0,0 +1,37 @@
+import ChatEntry from './ChatEntry';
+import PropTypes from 'prop-types';
+import './ChatLog.css';
+
+const ChatLog = ({entries, chatClicked}) => {
+  const chatComponents = entries.map((chat) => {
+    return (
+      <li key={chat.id}>
```

Comment:
Rather than wrapping our ChatEntry in a `li` here, another option could be to change `ChatEntry` so that it's outermost element was an `li` instead of a `div`